src/stravalib/model.py

```diff
@@ -336,33 +336,38 @@ class SummaryClub(MetaClub, strava_model.SummaryClub):
     dimensions: Optional[list[str]] = None
     localized_sport_type: Optional[str] = None
 
-    @lazy_property
-    def members(self) -> BatchedResultsIterator[Athlete]:
-        """
-        Lazy property to retrieve club members stored as Athlete objects.
-
-        Returns
-        -------
-        list
-            A list of club members stored as Athlete objects.
-        """
-        assert self.bound_client is not None, "Bound client is not set."
-        return self.bound_client.get_club_members(self.id)
-
-    @lazy_property
-    def activities(self) -> BatchedResultsIterator[Activity]:
-        """
-        Lazy property to retrieve club activities.
-
-        Returns
-        -------
-        Iterator
-            An iterator of Activity objects representing club activities.
-        """
-        assert self.bound_client is not None, "Bound client is not set."
-        return self.bound_client.get_club_activities(self.id)
-
-
+    # TODO: I don't understand where these lazy properties are called.
```

Comment:
@jsamoocha i'm not sure we need these lazy properties any more. The endpoints are separate and we are handling lazy loading with the batched iterator.  Please say the word if i'm missing something!